tests/test_bash_done_spam.py

```
@@ -0,0 +1,255 @@
+                bash.run(f"echo {marker}")
+
+                done_lines = JOB_DONE_RE.findall(bash.text)
+                if done_lines:
+                    raise AssertionError(
+                        "bash integration emitted job-completion "
+                        f"notifications ({len(done_lines)} line(s)). "
+                        "Issue: https://github.com/manaflow-ai/cmux/issues/1565\n\n"
+                        f"Done lines:\n{chr(10).join(done_lines)}\n\n"
+                        f"Bash: {bash_path}\n\n"
+                        f"Full PTY output:\n{bash.text}"
+                    )
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)